app/views/users/index.html.erb

```diff
@@ -1,12 +1,31 @@
 <section class="users-section">
+  
+
   <ul class="users-list">
     <% @users.each do |user| %>
       <li>
         Name: <%= user.name %>
         <span class="profile-link">
           <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+          <% if current_user.friend_invites(user) %>
+            <button class="btn btn-warning"> Invitation sent </button>
+          <% elsif current_user.receive_invitation(user)%>
+            <%= link_to 'Accept', accept_path(user_id: user.id), class:"btn btn-primary" %>
+            <%= link_to 'Decline', reject_path(user_id: user.id), method: :delete, class:"btn btn-danger" %>
+          <% elsif !current_user.friend?(user) && current_user.id != user.id %>
+            <%= link_to 'Add Friend', invite_path(user_id: user.id) , class:"btn btn-success" %>
+          <% elsif current_user.friend?(user)  %>
+            <button class="btn btn-warning"> A friend </button>
+          <% end %>
```

Comment:
Move logic into view helpers

Reply:
Logic has been moved to helpers.

Thanks